pkgs/top-level/kodi-packages.nix

```diff
@@ -171,4 +153,8 @@ let self = rec {
   trakt-module = callPackage ../applications/video/kodi/addons/trakt-module { };
 
   trakt = callPackage ../applications/video/kodi/addons/trakt { };
+
+  # deprecated or renamed packages
+
+  controllers = throw "kodi.packages.controllers has been replaced with kodi.packages.controller-topology-project - a package which contains a large number of controller profiles." { };
```

Comment:
That should be gated behind config.aliases.

Reply:
Do you mean `aliases.nix`? If so, I feel like this scenario is the same as our discussion [here](https://github.com/NixOS/nixpkgs/pull/117269#discussion_r599035419) - `aliases.nix` doesn't work for packages inside things like `kodi.packages`...

Reply:
I imagined to do it like this https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/gnome/default.nix#L283

Reply:
Great! Thanks for the feedback, I'll get that sorted today.

Reply:
Let me know if this works for you @SuperSandro2000. Thanks!